assets/docs/rishit-dagli/plant-disease/1.md

```diff
@@ -0,0 +1,35 @@
+# Module rishit-dagli/plant-disease/1
+Model to classify among 38 plant disease categories.
+
+<!-- module-type: image-classification -->
+<!-- network-architecture: mobilenet -->
+<!-- dataset: plant_village -->
+<!-- fine-tunable: false -->
+<!-- license: Apache-2.0 -->
+<!-- format: saved_model_2 -->
+<!-- asset-path: https://github.com/Rishit-dagli/Greenathon-Plant-AI/releases/download/v0.1.0/saved_model.tar.gz -->
```

Comment:
Hi @Rishit-dagli 
Thank you for addressing Morgans comments so quickly!
One more thing is that we cannot download the assets from the paths you provided since github.com/robots.txt forbids downloads from github.com/\*/download/*.
Could you upload the files to a different destination like GCS for example?
Thank you and sorry for the inconvenience!

Reply:
Hi @WGierke ,
Thanks for reviewing this PR. I am sorry for this. I will replace these with the GCS link.

Reply:
I have replaced the `SavedModel` and the model variants with the GCS link.